pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -371,6 +371,8 @@ in
     postPatch = ''
       substituteInPlace lib/rbreadline.rb \
         --replace 'infocmp' '${ncurses.dev}/bin/infocmp'
+
+      export HOME=$TEMPDIR
```

Comment:
Mhm, no idea what I tested actually, maybe your patch even work before?

Nevertheless, the following does now work. It is a bit more general
since it will also work if something else is using `metasploit-framework` in its dependency closure.


```diff
diff --git a/pkgs/development/ruby-modules/gem-config/default.nix b/pkgs/development/ruby-modules/gem-config/default.nix
index dc8643adec8..e7d1156acff 100644
--- a/pkgs/development/ruby-modules/gem-config/default.nix
+++ b/pkgs/development/ruby-modules/gem-config/default.nix
@@ -226,6 +226,12 @@ in
     '';
   };

+  metasploit-framework = attrs: {
+    preInstall = ''
+      export HOME=$TMPDIR
+    '';
+  };
+
   msgpack = attrs: {
     buildInputs = [ msgpack ];
   };
@@ -371,8 +377,6 @@ in
     postPatch = ''
       substituteInPlace lib/rbreadline.rb \
         --replace 'infocmp' '${ncurses.dev}/bin/infocmp'
-
-      export HOME=$TEMPDIR
     '';
   };
```